### PR TITLE
proto messages in vats

### PR DESCRIPTION
**Pull request by turadg**
refs: #9111

## Description

#9111 specified to support both encodings in `VLOCAL_CHAIN_EXECUTE_TX`. Instead, @michaelfig determined it would be better to have the one encoding of Protobuf 3 Any into JSON, and have userland utilities to convert any encoding into that. We considered supporting a binary encoding but those can't go over the bridge without hacks like the lower 8 bits of each character in a string.

To support the proto messages in vats, I had to kill imports that were assuming a browser/Node environment (e.g. axios that expected `http` and `stream`). We wanted to use those for off-chain clients but it's not necessary for Orchestration. It will take more work at some point: https://github.com/Agoric/agoric-sdk/issues/9200 

### Security Considerations

No change.

### Scaling Considerations

Larger bundles when the registry is imported.

### Documentation Considerations

We'll want to document these utilities as part of the Orchestration API.

### Testing Considerations

CI

### Upgrade Considerations

The only runtime change to a production vat is a validation that the list given to `executeTx` isn't empty. Even that isn't an _upgrade_ consideration because `vat-localchain` isn't deployed yet.

### CODE REVIEW

File: packages/casting/test/test-interpose-net-access.js [hunk 1]
```diff
@@ -69,21 +73,22 @@ const scenario2 = {
   ],
 };
 
-test(`vstorage query: Children (RECORDING: ${RECORDING})`, async t => {
+test.failing(`vstorage query: Children (RECORDING: ${RECORDING})`, async t => {
```
Comment (0xpatrickdev) [inline packages/casting/test/test-interpose-net-access.js]: Is this intentional?

File: packages/cosmic-proto/scripts/codegen.cjs [hunk 2]
```diff
@@ -86,13 +86,13 @@ telescope({
     },
     aminoEncoding: {
       // Necessary for getSigningAgoricClient
-      enabled: true,
+      enabled: false,
```
Comment (0xpatrickdev) [inline packages/cosmic-proto/scripts/codegen.cjs:89]: `dapp-inter` and other web clients are likely dependent on amino encodings. Agree they should be removed for `vats`, but something to make sure is in scope as part of #9200

**Review by 0xpatrickdev — APPROVED** [review]

File: packages/casting/test/test-interpose-net-access.js (hunk 1 above)
Comment (turadg) [reply packages/casting/test/test-interpose-net-access.js]: yes, punted to #9200

File: packages/cosmic-proto/scripts/codegen.cjs (hunk 2 above)
Comment (turadg) [reply packages/cosmic-proto/scripts/codegen.cjs:89]: Good callout, though it won't break until they try to upgrade past 0.4.0 and by then we may have #9200. If not then they shouldn't upgrade. I expect their CIs to fail if they upgrade to a version lacking the requisite functionality.

File: packages/agoric-cli/src/lib/chain.js [hunk 3]
```diff
@@ -96,16 +95,21 @@ harden(execSwingsetTransaction);
...
+export const fetchSwingsetParams = net => {
+  const { chainName, rpcAddrs } = net;
+  const cmd = [
+    `--node=${rpcAddrs[0]}`,
+    `--chain-id=${chainName}`,
+    'query',
+    'swingset',
+    'params',
+    '--output',
+    '--json',
```
Comment (michaelfig) [inline packages/agoric-cli/src/lib/chain.js]: `--json` is not a flag, it is an argument to `--output`, like `--output json` or `--output=json` or `-ojson` or `-o json`:
```suggestion
    'json',
```

File: packages/cosmic-proto/test/test-helpers.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+  const address = 'addr';
+  const qabr = typedJson('/cosmos.bank.v1beta1.QueryAllBalancesRequest', {
+    address,
+    // @ts-expect-error invalid field
+    other: 3,
+  });
+  t.deepEqual(qabr, {
+    '@type': '/cosmos.bank.v1beta1.QueryAllBalancesRequest',
+    address,
+    other: 3, // retained because there's no runtime validation
```
Comment (0xpatrickdev) [inline packages/cosmic-proto/test/test-helpers.js:29]: nb: we may want to consider using `.fromPartial()` to trim extra fields. It seems this is available via `registry`

**Pull request merged**